sentinel.go

```diff
@@ -317,6 +317,16 @@ func (c *SentinelClient) GetMasterAddrByName(ctx context.Context, name string) *
 	return cmd
 }
 
+func (c *SentinelClient) GetSlavesAddrByName(ctx context.Context, name string) []string {
```

Comment:
I am going to unexport this since it looks like a helper function. Let me know if you need it.

Reply:
Yes, it should be in the test file